ddl/ddl_api.go

```diff
@@ -4372,6 +4372,10 @@ func (d *ddl) AlterTableSetTiFlashReplica(ctx sessionctx.Context, ident ast.Iden
 	if err != nil {
 		return errors.Trace(err)
 	}
+	// Ban setting replica count for tables in system database.
+	if util.IsMemOrSysDB(schema.Name.L) {
+		return errors.Trace(errUnsupportedAlterReplicaForSysTable)
```

Comment:
I think no need to trace it.

Reply:
why? trace is good for locating the code in the user log in different tidb version.